### DISCUSSION TITLE
createShopReturnPolicy

### DISCUSSION THREAD

**Discussion by amitkumar-248** [etsy/open-api#767] [Q&A] [answered]
Hello Team, 
we are getting errors with API when creating a new return policy that do not accept returns or exchanges 
according to [documentation](https://developers.etsy.com/documentation/reference#operation/createShopReturnPolicy) we can pass null value to **return_deadline**
error response as 
[error] => Missing input parameter: [return_deadline]

header response 
 [X-Etsy-Request-Uuid] => Array
                        (
                            [0] => EubM7wKzR3kMzj15DmX-Sh3SHa1c
                        )
                        
Can someone help us to solve this problem?
thanks

**Comment by etsyachristensen** [comment]
@amitshuklaCedcoss are you providing the `return_deadline` input, but leaving it null? Or not including it at all?

**Reply by amitkumar-248** [reply]
hi @etsyachristensen 
I tried with all cases like 
case 1: return_deadline : null  **[error] => Missing input parameter: [return_deadline]**  
                      [X-Etsy-Request-Uuid] => Array
                        (
                            [0] => EuJaaOhxhbUK_hz7B0iR1FvWUy6d
                        )
case 2: return_deadline :0  **[error] => Shop Policy Data is invalid**
case 3: not adding this index in post data **("error": "Missing input parameter: [return_deadline]")**

**Comment by amitkumar-248** [comment, marked as answer]
hi @etsyachristensen 
now doc updated and concern resolved. 
thanks
